samples/githubDispatch/command.js

```diff
@@ -0,0 +1,62 @@
+const { Octokit } = require("octokit");
+/**
+* This is the implementation of your Fusebot command.
+* You can run it from Slack with `/fusebot trigger-fusebit-pipeline ...`.
+* Fusebot is powered by the Fusebit integration platform, https://fusebit.io/
+* 
+* Docs and samples are at https://github.com/fusebit/fusebot
+* You can talk to a fellow developer on Slack at https://fusebit.io/contact
+* Or tell us what you think with `/fusebot feedback {your-comments}` 
+*
+* Now let's build something already. 
+
+* @param ctx {FusebotContext}
+*/
+module.exports = async (ctx) => {
+  if (ctx.body.args.length === 0 || ctx.body.args[0] === "help") {
+    return help(ctx);
+  }
+  if (!ctx.configuration.PAT) {
+    await ctx.client.send(
+      "unable to find github PAT in configuration, please set a PAT in configuration with the scope of * on repo and write on action."
+    );
+  }
+  const octokit = new Octokit({ auth: ctx.configuration.PAT });
+  const [userRepo, workflowId, branchTagName] = ctx.body.args;
+  let isError = false;
+  if (!userRepo) {
+    isError = true;
+    await ctx.client.send("Username/repository was not found.");
+  }
+  if (!workflowId) {
+    isError = true;
+    await ctx.client.send("WorkflowId/workflowFileName was not found.");
+  }
+  if (!branchTagName) {
+    isError = true;
+    await ctx.client.send("Branch name or tag name was not found.");
+  }
```

Comment:
How about you follow this up with the text of help itself, so that folks can see for themselves what the expected usage of the command is without extra steps? Something like 

```
if (!userRepo) {
  return await help(ctx, "You must specify the username/repository parameter");
}
```

Then: 

```
async help(ctx, optionalMessage) {
  const message = [];
  if (optionalMessage) {
    message.push(optionalMessage);
  }
  message.push(...regularHelp...);
  ctx.client.send(message.join('\n');
}
```

Reply:
Done